contrib/ros/worlds/empty_world/world.py

```diff
@@ -0,0 +1,43 @@
+import collections
```

Comment:
There's no need to create a whole package subfolder for a single module.

Just put it in `worlds/empty_world.py`